snmpwalk/manifest.json

```diff
@@ -1,18 +1,15 @@
 {
   "maintainer": "help@datadoghq.com",
-  "manifest_version": "0.1.0",
-  "max_agent_version": "6.0.0",
-  "min_agent_version": "5.6.3",
+  "manifest_version": "1.0.0",
```

Comment:
why does the version here need to change?

Reply:
number and type of fields changed in `1.0.0`